samples/drivers/led_pwm/sample.yaml

```
@@ -13,11 +13,10 @@ tests:
       type: multi_line
       ordered: true
       regex:
-        - "Found device LED_PWM_\\d+"
+        - "Found device [^\\s]"
```

```suggestion
        - "Found device \\S+"
```
To match one or more non-whitespace characters at the end of the line.